Added some new commands

MakerPact opened the pull request:
Added new commands to the bot along with updating the readme, and also put all the tags in alphabetical order.
@max-bromberg

arctfoxx commented:
Awesome! Just please remember the conventions laid out to keep PRs consistent Thanks!

MakerPact committed:
Adds more tag options for Arduino help

Extends the available tag commands with more options
related to Arduino development.

Adds tags for level shifters, debouncing, analog read,
motor control, and PWM.

MakerPact committed:
Adds 'wiki' to tag options

Adds 'wiki' to the list of available tag options.

MakerPact committed:
Add wiki section with resources for Arduino basics

MakerPact committed:
Adding wikii tag.

MakerPact committed:
Merge branch 'staging' into WokwiiTag

MakerPact committed:
Merge pull request #3 from MaderDash:WokwiiTag

Add wiki section with resources for Arduino basics

MakerPact committed:
Merge branch 'arduinodiscord:staging' into staging

MakerPact committed:
Revert "Merge pull request #3 from MaderDash:WokwiiTag"

This reverts commit d5fe7da193adf354b439f791ab923342adf64b25, reversing
changes made to 700418c4c7b1603154dd508d9fc7fc72c3a9d173.

MakerPact committed:
Add wiki embed with Arduino basics information and the command so that it should be woring.

MakerPact committed:
Added bold to the links, for the drivers.

MakerPact committed:
Merge branch 'Wikii' into staging

MakerPact committed:
Add level shifter tag with detailed information and embed. This should give us a levelshifter tag.

MakerPact committed:
Update README.md to include detailed bot description and current slash commands, and a general overview of how to add tags and commands, and also how to create an issue.

MakerPact committed:
Enhance Arduino IDE documentation with additional details on programming languages, library inclusion, and voltage mismatch issues.

MakerPact committed:
Refactor tag responses for clarity and consistency; enhance logic level shifter and pull-up explanations.
Also ordered the tags in alphabetical order so we can find them better.

MakerPact committed:
Add wiki embed with Arduino basics information and the command so that it should be woring.

MakerPact committed:
Adding wikii tag.

MakerPact committed:
Added bold to the links, for the drivers.

MakerPact committed:
Enhance Arduino IDE documentation with additional details on programming languages, library inclusion, and voltage mismatch issues.

MakerPact committed:
Update README.md to include detailed bot description and current slash commands, and a general overview of how to add tags and commands, and also how to create an issue.

MakerPact committed:
Add level shifter tag with detailed information and embed. This should give us a levelshifter tag.

MakerPact committed:
Revise help prompts in tags.ts for clarity and effectiveness, enhancing user guidance on asking questions and troubleshooting common issues.

MakerPact committed:
Add help prompts for effective assistance and enhance Arduino wiki information

MakerPact committed:
Merge branch 'staging' of https://github.com/MaderDash/bot into staging

MakerPact committed:
fixing the doubble "'s so that the formatting should work better.

MakerPact committed:
Reorder 'wiki' and 'levelShifter' entries in tag command options for consistency

MakerPact committed:
Enhance help prompts in tags.ts for clarity and detail, improving user guidance on troubleshooting and Arduino usage.

MakerPact committed:
Merge branch 'staging' of https://github.com/MaderDash/bot into staging

MakerPact committed:
Reorder tag command options for consistency and improve help descriptions in tags.ts for clarity
